recipes/psy-strat/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "psy-strat" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "b68aa45522cf5cab5fb270f78216f6ed45ea03d0be0678c7b7419c3048191baf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed -vvv
+  skip: true  # [py27]
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy>=1.14  # [win]
```

Comment:
You need spaces here between the package name and the version, eg `numpy >=1.14`.

Also, the way this is written numpy is not a runtime dependence for osx or linux. I don't think this is what you want. To fix this, add another dep like this `numpy  # [not win]`

Reply:
Thanks @beckermr! Actually this run dependency is due to some incompatibilities between dask and numpy. I frequently encountered an issue on appveyor (see for example https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/22138939) where the import of dask leads to

`AttributeError: module 'numpy' has no attribute 'AxisError'`

This happens because numpy is chosen to be 1.11 for some reasons. Requiring `numpy >=1.14` was the simplest solution that came to my mind but I am open for your suggestions.

Reply:
Ahhh. So this needs to be filed as an issue with dask instead of patching the dependence here.

Reply:
Hey @beckermr! After having another look into it, I realized that the problems actually came from the `xarray` package, not `dask`. And this issue has already been adressed with `xarray >=0.11` but for some reasons, conda installed `xarray=0.10.9` on appveyor. 

So what is the recommended solution here? Should I require `xarray >=0.11` for `straditize` and `psy-strat`? Or `numpy >=1.12`? But this is actually just a problem of the `xarray` conda-package, not an incompatibility of `straditize`, etc..

Reply:
This is your choice. A constraint on a version is fine by me.